microshift_updating/microshift-update-options.adoc

```diff
@@ -48,10 +55,19 @@ To understand more about Greenboot, see the following documentation:
 
 [id="microshift-update-options-manual-rpm-updates_{context}"]
 === Manual RPM updates
-You can update {microshift-short} manually on a non-OSTree system such as {op-system-base-full} by downloading and updating the RPMs. To complete this update type, use the subscription manager to access the repository containing the new RPMs. To begin a manual RPM update, use the procedures in the following documentation:
+You can update {microshift-short} manually on a non-OSTree system such as {op-system-base-full} by downloading and updating the RPMs.
```

Comment:
Is the `downloading the RPMs` a catchall phrase ? It is not necessarily required as long as you are properly pointing to the correct repo. You can run a simple dnf upgrade <microshift-version> and the package will be upgraded

Reply:
No, that's a carryover from 4.12, which was dev preview so there was no subscription repo yet. Good catch, TY!